src/main/java/uk/gov/hmcts/darts/event/service/impl/DarNotifyServiceImpl.java

```diff
@@ -2,36 +2,23 @@
 
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.event.EventListener;
 import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Service;
-import uk.gov.hmcts.darts.common.entity.NodeRegisterEntity;
 import uk.gov.hmcts.darts.common.repository.CaseRepository;
-import uk.gov.hmcts.darts.common.repository.NodeRegisterRepository;
-import uk.gov.hmcts.darts.event.client.DartsGatewayClient;
 import uk.gov.hmcts.darts.event.exception.DarNotifyError;
+import uk.gov.hmcts.darts.event.helper.DarNotifyAsyncHelper;
 import uk.gov.hmcts.darts.event.model.DarNotifyApplicationEvent;
-import uk.gov.hmcts.darts.event.model.DarNotifyEvent;
-import uk.gov.hmcts.darts.log.api.LogApi;
 
-import java.text.MessageFormat;
 import java.util.List;
-import java.util.Locale;
-import java.util.Optional;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
 public class DarNotifyServiceImpl {
 
-    private final DartsGatewayClient dartsGatewayClient;
     private final CaseRepository caseRepository;
-    private final NodeRegisterRepository nodeRegisterRepository;
-    private final LogApi logApi;
-
-    @Value("${darts.dar-pc-notification.url-format}")
-    private String notificationUrlFormat;
+    private final DarNotifyAsyncHelper darNotifyAsyncHelper;
 
     @Async
```

Comment:
Probably need to remove this `@Async`?

Reply:
as discussed, this is existing functionality which we are going to leave for now.